core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/Stain.java

```diff
@@ -0,0 +1,42 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+import java.io.Serializable;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "Stain")
+
+public class Stain implements Serializable {
+  private static final long serialVersionUID = 1L;
+  private String stainGroup;
```

Comment:
Probably neither. We are going to get a whole bunch of stains that come in different group (especially antibodies), so the code sorts them into groups based on these labels. Maybe we can do submenus.